src/Modal/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Modal as AntdModal } from 'antd';
+import { ModalProps as AntdModalProps } from 'antd/es/modal';
+import React, { PropsWithChildren } from 'react';
+
+export const Modal: typeof AntdModal = AntdModal;
+export type ModalProps = AntdModalProps;
+
+export function FullScreenModal({
+  children,
+  style,
+  ...props
+}: PropsWithChildren<AntdModalProps>) {
```

Comment:
```suggestion
}: PropsWithChildren<ModalProps>) {
```

In case we modify the base `Modal`.